config/deploy.rb

```diff
@@ -46,36 +46,20 @@
 namespace :sneakers do
   task :restart do
     on roles(:worker) do
-      case fetch(:stage)
-      when "production"
-        execute :sudo, :service, "dpul-sneakers", :restart
-      when "staging"
-        execute :sudo, :initctl, :restart, "pom-sneakers"
-      end
+      execute :sudo, :service, "dpul-sneakers", :restart
     end
   end
 end
 
 namespace :sidekiq do
   task :quiet do
     on roles(:worker) do
-      # Horrible hack to get PID without having to use terrible PID files
```

Comment:
I think this comment was here before I added the conditional; maybe it should be restored?